types/test/model-count.ts

```diff
@@ -4,7 +4,7 @@ import { Model, Op } from 'sequelize';
 class MyModel extends Model {}
 
 expectTypeOf(MyModel.count()).toEqualTypeOf<Promise<number>>();
-expectTypeOf(MyModel.count({ group: 'tag' })).toEqualTypeOf<Promise<{ [key: string]: number }>>();
+expectTypeOf(MyModel.count({ group: 'tag' })).toEqualTypeOf<Promise<({ [key: string]: any } & { count: number })[]>>();
```

Comment:
```suggestion
expectTypeOf(MyModel.count({ group: 'tag' })).toEqualTypeOf<Promise<({ [key: string]: unknown, count: number })[]>>();
```

The important part being replacing `any` with `unknown` but can't we merge these two types like this?